### ISSUE TITLE
Higher fidelity foreground and background descriptions

### ISSUE THREAD

**Issue by meowgorithm** [charmbracelet/sequin#16]
It could be helpful to have a little more detail in color sequences. For example, the following sequences currently translate as…

```
\x1b[32m     -> CSI 32m:    Foreground color: Green
\x1b[38;5;4m -> CSI 38;5;4m Foreground color: 4
```

…and it could be nice to see:

```
\x1b[32m     -> CSI 32m:    ANSI foreground color: Green
\x1b[38;5;4m -> CSI 38;5;4m ANSI256 foreground color: 4
```

**Comment by mkotowski** [comment]
Hi, I created PR #40 with a proposition based on changes suggested here (+ small fix for SGR 59 and slight change to display of the RGB values). Please take a look and let me know if this is acceptable. :slightly_smiling_face:

**Comment by caarlos0** [comment]
fixed by #40 

thanks so much @mkotowski!
